packages/docs/src/pages/tutorial/store/no-serialize/index.mdx

```diff
@@ -3,6 +3,8 @@ title: Non-serializable Properties
 layout: tutorial
 ---
 
-Stores must always be serializable. However, there are times when it is useful to store a value on a store that is not serializable. Such a value will persist only while the application is resumed. If the application is paused the value will be thrown away. (This is not an idiomatic way to do this, only a contrived example.)
+Stores must always be serializable, however sometimes you may want persist a value on a store that can't be serialized. Values like these persist in the browser only when the application is resumed on the client. 
 
-In this example when the clock is started we need to save a cleanup function in `store.cleanup` so that the `stop` button can clear the interval.
\ No newline at end of file
+When the application is paused, and execution returns to the server, non-serializable properties are discarded.
```

Comment:
What do you mean by "execusion returns to the server"? 

AFAIK, execusion starts on the server and then getting "paused" (serialized) and then resumes on the client.
Another server call is either ajax or a full page load (which will not continue, just start over from a different entry point)

@mhevery can you verify that what I've written here is correct?

Reply:
That makes sense. Thanks for clarifying my thinking here. I'll wait for Misko to chime in, but I can certainly adjust accordingly.

Reply:
@mhevery did you see this? :)

Reply:
`and execution returns to the server` does not make sense. I think you want to write:
```
When the application is paused, non-serializable properties are discarded.
```

Reply:
This is now fixed!